src/containers/ViewerPanel/index.tsx

```diff
@@ -431,6 +440,41 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
         }
     };
 
+    private get embedDisplaySettings() {
+        const { height, width } = this.state;
+        const belowControlsHeight = height <= CONTROLS_MIN_HEIGHT;
+        const belowControlsWidth = width <= CONTROLS_MIN_WIDTH;
+        const showScaleBar =
+            width > SCALE_BAR_MIN_WIDTH &&
+            (!belowControlsWidth || !belowControlsHeight);
```

Comment:
this is still too confusing. Is there a cutoff where you hide this? 
Ie can you have `const hideScaleBar < SCALE_BAR_MIN_WIDTH`?
Otherwise you can set it in the conditionals below. ie line 455: `showScaleBar: false` 
line 461 and 467 `showScaleBar: width > SCALE_BAR_MIN_WIDTH`. line 473: `showScaleBar: true`

Reply:
The point of these checks is to make sure the scale bar doesn't overlap any controls. The height is relevant because when we drop below that breakpoint, we move the home button into the bottom controls, which changes the width occupied by the bottom controls.

So if we are
below scale bar cutoff: false
above scale bar cut off, above the width control cut off, we have room for scale bar: true
above scale bar cut off, and below width cutoff, we need to know if the bottom controls are padded by the presence of the home button (i.e. check the height cutoff)

Anyway, I like your last suggestion so I'm going to go with that!